app/build.gradle

```diff
@@ -155,6 +159,28 @@ tasks.withType(Test) {
     ]
 }
 
+task buildTranslationArray {
+    def foundLocales = new StringBuilder()
```

Comment:
What's the usage of this? I'm quite curious.

Reply:
To synchronize the resource-file `locales_config.xml` with the code. I know that this is a really big hack but honestly I have spent to much time searching for a better way of doing it and it works so...

Reply:
Basically it is so that we only show to the user language that we have a translation for